index.html

```diff
@@ -57,7 +57,7 @@
 				{
 				name: "Michael Cooper",
 				company: "W3C",
-				companyURL: "http://www.w3.org",
+				companyURL: "https://www.w3.org",
```

Comment:
So I thought this and some of the other links (around AAPIs) was already changed in https://github.com/w3c/aria/pull/2102/ 

But now I see that that @daniel-montalvo 's PR went into the 1.3 FWPD branch, not main

This will cause merge issues down the line.

@daniel-montalvo can you help rebase things?